### PR TITLE
Various fixes and additions

### PR DESCRIPTION
**Pull request by projectdp**
- Fixed markdown issues per markdown linter.
- Moved awesome repositories into the correct section and moved non-awesome repositories to the correct section.
- Sorted lists.
- Added a variety of repositories.
- Added Track Awesome List.
- Added Awesome button.
- Minor fixes to contributions, added self to contributions.

### CODE REVIEW

**Comment by cat998** [comment]
已收到！

**Comment by Fjrhamzah277** [comment]
saya tertarik mempelajari membuat applikasi peretasan Android dengan cara umpan dengan Link. apakah ada yang bisa membantu?

**Review by Caoxokhuyen — APPROVED** [review]
def fibonacci(n)
  return n if ( 0..1 ).include? n
  (fibonacci(n - 1) + fibonacci(n - 2)) #recursive calls
end

**Comment by Ethical1and0** [comment]
Great.

**Comment by LancealotSeven** [comment]
///////AA:A////HUT DOWN THE BUG////AA:A////AL////AA:G////O ////AA:A////HUT DOWN THY HACKER///////LL:L///////FROM TAKING DOWN YOUR FIREWALL///////

**Comment by Pawancyber** [comment]
Nice.

**Comment by cat998** [comment]
已收到！

**Comment by IAmBlackHacker** [comment]
❤️

**Comment by 9miuz** [comment]
Thanks

**Review by ohhruna — APPROVED** [review]

**Review by Leyder2007 — CHANGES_REQUESTED** [review]
Viperbleis

**Comment by Chan9390** [comment]
@projectdp This is cool. Can you please remove the badges?

**Comment by Kishore007raj** [comment]
cool!!

**Review by qwerrttz234 — APPROVED** [review]

**Review by NazarTovstyha1 — APPROVED** [review]

**Review by Lilitmrz — COMMENTED** [review]
9

**Review by Lilitmrz — CHANGES_REQUESTED** [review]
H

File: contributing.md [hunk 1]
```diff
@@ -21,12 +21,13 @@ This resource was made by the developers and hackers alike! We appreciate and re
 
 ## Contributors
 
+- [Alan Chang](https://github.com/tcode2k16)
+- [Aleksandar Todorović](https://github.com/aleksandar-todorovic)
 - [Chandrapal](https://github.com/Chan9390)
-- [Madhu Akula](https://www.github.com/madhuakula)
 - [Derick Thomson](https://www.facebook.com/derick.thomson) (Image)
```
Comment (Leyder9537thd) [inline contributing.md:27]: ] ( https://github.com/infoslack/awesome-web-hacking )

**Review by Leyder9537thd — CHANGES_REQUESTED** [review]

**Review by Femida666 — APPROVED** [review]

**Review by devilish66694 — APPROVED** [review]
Good

**Review by devilish66694 — APPROVED** [review]

**Review by llkkkklllllls — APPROVED** [review]

**Review by llkkkklllllls — APPROVED** [review]

**Review by llkkkklllllls — APPROVED** [review]

File: contributing.md [hunk 2]
```diff
@@ -21,12 +21,13 @@ This resource was made by the developers and hackers alike! We appreciate and re
...
-- [Aleksandar Todorović](https://github.com/aleksandar-todorovic)
-- [Sobolev Nikita](https://github.com/sobolevn)
-- [Serhii Pronin](https://github.com/re-pronin)
+- [Madhu Akula](https://www.github.com/madhuakula)
+- [Projectdp](https://www.github.com/projectdp)
 - [ReadmeCritic](https://github.com/ReadmeCritic)
-- [Yakup Ateş](https://github.com/y-ates)
-- [Alan Chang](https://github.com/tcode2k16)
```
Comment (llkkkklllllls) [inline contributing.md:32]: (@IAmBlackHacker _**_)

**Review by llkkkklllllls — APPROVED** [review]

**Review by devilish66694 — APPROVED** [review]

File: contributing.md (hunk 2 above)
Comment (devilish66694) [reply contributing.md:32]: > (@IAmBlackHacker _**_)

Verified

File: contributing.md (hunk 1 above)
Comment (devilish66694) [reply contributing.md:27]: > ] ( https://github.com/infoslack/awesome-web-hacking )

**Review by devilish66694 — APPROVED** [review]

**Review by devilish66694 — APPROVED** [review]

**Review by w1s3one805 — APPROVED** [review]

**Comment by Fjrhamzah277** [comment]
I have not received

Pada Kam, 27 Jun 2024 12.06, w1s3one805 ***@***.***> menulis:

**Review by IAmBlackHacker — APPROVED** [review]

**Comment by Chan9390** [comment]
There are quite a few conflicts with existing state. I'm closing the PR for now. Feel free to create new one :)

**Pull request closed without merging**